front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleModal.tsx

```diff
@@ -319,19 +366,27 @@ const ImportTrainScheduleModal = ({
           {!infraId || !timetableId || !rollingStockID ? null : (
             <>
               <button
-                className={`btn btn-sm btn-block d-flex justify-content-between ${
-                  status.uicComplete ? 'btn-outline-success' : 'btn-primary'
-                }`}
+                className={cx('btn btn-sm btn-block d-flex justify-content-between', {
```

Comment:
Not the best case, but since the front meeting, should we isolate each of these class when using a `cx` ?
```suggestion
                className={cx('btn', 'btn-sm', 'btn-block', 'd-flex', 'justify-content-between', {
```

Reply:
Yes, well spotted, thanks !